sources/packages/web/src/constants/system-constants.ts

```diff
@@ -1,15 +1,19 @@
 // * MINIMUM_TOKEN_VALIDITY in seconds, i,e 180 seconds is 3 minutes
 export const MINIMUM_TOKEN_VALIDITY = 180;
 // TODO : for development  added all IDLE timeout as 1hour, revert it back to 4.5 before merging to PROD
-// * MINIMUM_IDLE_TIME_FOR_WARNING_STUDENT in seconds, i,e 270 seconds is 4.5 minutes
-export const MINIMUM_IDLE_TIME_FOR_WARNING_STUDENT = 3600;
-// * MINIMUM_IDLE_TIME_FOR_WARNING_STUDENT in seconds i,e 270 seconds is 4.5 minutes
-export const MINIMUM_IDLE_TIME_FOR_WARNING_INSTITUTION = 3600;
-// * MINIMUM_IDLE_TIME_FOR_WARNING_STUDENT in seconds i,e 270 seconds is 4.5 minutes
-export const MINIMUM_IDLE_TIME_FOR_WARNING_SUPPORTING_USER = 3600;
-// * MINIMUM_IDLE_TIME_FOR_WARNING_STUDENT in seconds i,e 270 seconds is 4.5 minutes
-export const MINIMUM_IDLE_TIME_FOR_WARNING_AEST = 3600;
+// * MAXIMUM_IDLE_TIME_FOR_WARNING_STUDENT in seconds, i,e 270 seconds is 4.5 minutes
```

Comment:
May be not a part of PR, but these values can come from environment.

Reply:
Are we going to have a technical debt for this. ?

Reply:
@andrewsignori-aot @ann-aot @guru-aot Should we create a ticket for this?

Reply:
ya. I belive our plan was to create env variable in the future

Reply:
I do not see it as a priority right now. Can be useful for testing purposes only, IMO.